src/components/sbb-option/sbb-option.tsx

```diff
@@ -0,0 +1,268 @@
+ * @slot unnamed - Use this to provide the option label.
+ * @slot icon - Use this slot to provide an icon. If `icon-name` is set, a sbb-icon will be used.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-option.scss',
+  tag: 'sbb-option',
+})
+export class SbbOption implements ComponentInterface {
+  /** Value of the option. */
+  @Prop() public value?: string;
+
+  /**
+   * The icon name we want to use, choose from the small icon variants
+   * from the ui-icons category from here
+   * https://lyne.sbb.ch/tokens/icons/.
+   */
+  @Prop() public iconName?: string;
+
+  /** Whether the option is currently active. */
+  @Prop({ reflect: true }) public active?: boolean;
+
+  /** Whether the option is selected. */
+  @Prop({ mutable: true, reflect: true }) public selected = false;
+
+  /** Whether the component must be set disabled due disabled attribute on sbb-checkbox-group. */
+  @State() private _disabledFromGroup = false;
+
+  /** Whether the option is disabled. TBI: missing disabled style, will be implemented with the select component. */
+  @Prop() public disabled?: boolean;
+
+  /** Emits when the option selection status changes. */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'option-selection-change',
+  })
+  public selectionChange: EventEmitter<SbbOptionEventData>;
+
+  /** Emits when the option is clicked. */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'option-click',
+  })
+  public optionClick: EventEmitter<SbbOptionEventData>;
+
+  /** State of listed named slots, by indicating whether any element for a named slot is defined. */
+  @State() private _namedSlots = createNamedSlotState('icon');
+
+  @State() private _label: string;
+
+  /** The portion of the highlighted label. */
+  @State() private _highlightString: string;
+
+  /** Disable the highlight of the label. */
+  @State() private _disableLabelHighlight: boolean;
+
+  @State() private _groupLabel: string;
+
+  @Element() private _element!: HTMLElement;
+
+  private _handlerRepository = new HandlerRepository(
+    this._element,
+    namedSlotChangeHandlerAspect((m) => (this._namedSlots = m(this._namedSlots)))
+  );
+
+  private _optionId = `sbb-option-${++nextId}`;
+  private _variant: SbbOptionVariant;
+  private _inertAriaGroups = isSafari();
+
+  /** MutationObserver on data attributes. */
+  private _optionAttributeObserver = new MutationObserver(
+    this._onOptionAttributesChange.bind(this)
+  );
+
+  /**
+   * Highlight the label of the option
+   * @param value the highlighted portion of the label
+   */
+  @Method()
+  public async highlight(value: string): Promise<void> {
+    this._highlightString = value;
+  }
+
+  /**
+   * Set the option group label (used for a11y)
+   * @param value the label of the option group
+   */
+  @Method()
+  public async setGroupLabel(value: string): Promise<void> {
+    this._groupLabel = value;
+  }
+
+  @Listen('click', { passive: true })
+  public selectByClick(): void {
```

Comment:
Initially, it was implemented for a specific internal need. 
I thought it might be useful for customers but I'm not sure anymore.
You might be right, we could easily do without it.

Reply:
Remove the option-click event